frontend/src/pages/Help.tsx

```diff
@@ -0,0 +1,187 @@
+          usual procedure for logging time with Redmine involves selecting an
+          issue and an activity for reporting your time. In urdr, issues are
+          strictly linked to valid activities, thus making it simpler to do time
+          reporting accurately. In order to improve the user experience, the
+          time reporting page offers a spreadsheet-like grid for time entry
+          reporting, together with an intuitive way of navigating over different
+          weeks. The report view is primarily made up of a header containing a
+          week navigation handler and a user dropdown menu, the time entries
+          grid, and the bottom bar including the feature for adding new rows and
+          saving your changes.
+        </p>
+        <div className="centered">
+          <img src={overview} alt="overview of urdr" className="overview-img" />
+        </div>
+        <h2 className="help-subtitle">What is a row?</h2>
+        <p className="help-info">
+          Within the context of the urdr system, we refer to a row as to the
+          visual element on the page where the time entries of a given
+          issue-acivity pair are displayed for one specific week. There are two
+          types of rows: favourite and recent rows.
+        </p>
+        <h2 className="help-subtitle">Favourite and recent rows</h2>
+        <p className="help-info">
+          The following row has been marked as favourite so it will always
+          appear at the top of the list. Rows are marked as favourite /
+          non-favourite after clicking on the star button present next to the
+          title of the corresponding issue. It's worth noting that favourite
+          rows are saved across sessions. Moreover, the order of the favourite
+          rows can be changed by drag and dropping the row using the
+          drag–and–drop handle icon at the left-most position of the row.
+        </p>
+        <Row
+          topic={exampleIAP}
+          days={[new Date()]}
+          rowHours={[1]}
+          rowEntries={[exampleEntry]}
+          getRowSum={() => 1}
+          onHide={() => {}}
+          isFav={true}
+          onToggleFav={() => {}}
+          onCellUpdate={() => {}}
+        ></Row>
+        <p className="help-info">
+          The next one is a recent row. If desired, these type of rows can be
+          hidden and/or after clicking on the eye button. It's worth noting that
+          recent rows change as you navigate across different weeks or as you
+          add new time entries.
+        </p>
+        <Row
+          topic={exampleIAP}
+          days={[new Date()]}
+          rowHours={[1]}
+          rowEntries={[exampleEntry]}
+          getRowSum={() => 1}
+          onHide={() => {}}
+          isFav={false}
+          onToggleFav={() => {}}
+          onCellUpdate={() => {}}
+        ></Row>
+        <h2 className="help-subtitle">Adding new rows</h2>
```

Comment:
In between this sections, I would add a short explanation of the sum row below the recent time entries, maybe with a screenshot. Something like this: 

"A cell in the sum row shows the sum of all time entries logged on different rows on the corresponding day. It is worth noting that the sum also contains entries that have been hidden from list of recent rows. In case the number displayed in a sum field doesn't seem to fit to the time entries displayed above, you have probably hidden a row that contained a time entry for that day. You can easily make it visible again  by using the "Add new issue" feature (see below)."